nextjs-task-system/pages/api/tasks/[id].ts

```
@@ -0,0 +1,102 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import prisma from "../../../prisma/client";
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  const id = req.query.id ? parseInt(req.query.id as string, 10) : null;
+
+  if (!id || isNaN(id)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)